.gitignore

```diff
@@ -7,4 +7,5 @@ __pycache__
 *.pyo
 *.swp
 .pytest_cache/
+.mypy_cache/
```

Comment:
this seems unrelated?

Reply:
Yes, unrelated. Mypy was able to pinpoint syntaxerrors codestyle did not find.

Reply:
I'd like to keep it anyways.

Reply:
fine with me